sos/report/plugins/auditd.py

```diff
@@ -19,19 +19,20 @@ class Auditd(Plugin, IndependentPlugin):
     packages = ('audit',)
 
     def setup(self):
+        self.add_copy_spec("/etc/audit/auditd.conf", tags="auditd_conf")
         self.add_copy_spec([
-            "/etc/audit/auditd.conf",
```

Comment:
Should already be tagged as such:

```
{
                            "specification": "/etc/audit/auditd.conf",
                            "files_copied": [
                                "etc/audit/auditd.conf"
                            ],
                            "tags": [
                                "auditd_conf"
                            ]
                        },
```

Reply:
Oh are files tag automatically now?

Reply:
Yeah, but it's limited to the file name, not the whole path. So the above becomes `auditd_conf` and not `audit_auditd_conf`.

So, for the changes that make directory names part of the tag, keep those.

Reply:
Ah okay thanks I'll work on correcting those.